test/models/ProgramLocation

```diff
@@ -0,0 +1,36 @@
+const assert = require('assert');
```

Comment:
This file is missing the `.js` file extension so it is not being detected and run as part of the test suite.